bitcoin/build.rs

```diff
@@ -26,8 +24,13 @@ fn main() {
         .parse::<u64>()
         .expect("invalid Rust minor version");
 
+    let msrv = std::env::var_os("CARGO_PKG_RUST_VERSION").unwrap();
+    let mut msrv_semvar = msrv.to_str().unwrap().split(".");
```

Comment:
It's better to just call `std::env::var` since then we should get better error messages.

Also why `_semvar`? Beside strange suffix, I think it'd be simpler to just call it `msrv`.

Reply:
> Also why _semvar? Beside strange suffix, I think it'd be simpler to just call it msrv.

I'm not picky,  Will name it msrv if that's more clear.  Semvar to me was just a way of noting it had major, minor and patch version numbers.

Reply:
> It's better to just call std::env::var since then we should get better error messages.

The difference explained in the docs is that var will panic on invalid utf8.  I'll update it to `env::var` if you prefer.

Reply:
~The difference explained in the docs is that var will panic on invalid utf8. I'll update it to env::var if you prefer.~
 
sorry that's env::vars not env::var

Reply:
> Semvar to me was just a way of noting it had major, minor and patch version numbers.

Never heard the term used for that. Any version number has those components. Also referring to it as "semantic variable" sounds like talking about variable.

Reply:
> Never heard the term used for that. Any version number has those components. Also referring to it as "semantic variable" sounds like talking about variable.

What I meant to type was semver not samvar which maybe is what you're pointing out.  Anyway, the assignment has been renamed to avoid confusion.